Src/coffeescript/cql-execution/test/datatypes/datetime.coffee

```diff
@@ -396,6 +396,15 @@ describe 'DateTime.differenceBetween', ->
     b = DateTime.parse '2012-07-14T23:59:00.0+00:00'
     a.differenceBetween(b, DateTime.Unit.DAY).should.eql new Uncertainty(121)
     a.durationBetween(b, DateTime.Unit.DAY).should.eql new Uncertainty(121)
+    a = DateTime.parse '2012-09-13T14:50:00.0-04:00'
+    b = DateTime.parse '2012-12-31T23:59:59.999-05:00'
+    a.differenceBetween(b, DateTime.Unit.MONTH).should.eql new Uncertainty(3)
+    a.durationBetween(b, DateTime.Unit.MONTH).should.eql new Uncertainty(3)
+    # The following test, for some reason, is BROKEN!
```

Comment:
@cmoesel says this comment can be removed now, as this was addressed by a previous change.  The test should be working now.

Reply:
Removed.